[BUG] npm run test failed due to lack of permission

# Environment
OS: `Ubuntu 16.04.6 LTS`
npx version: `6.4.1`
nvm version: `0.34.0`
node version: `v10.15.1`
npm version: `6.4.1`

# Problem
When executing `npm run test` after unboxing the `npm-truffle-box` (using `npx truffle unbox wanseob/npm-truffle-box`), operation failed with the message; `sh: 1: ./scripts/test.sh: Permission denied`

Below are permission of related files.
```
$ ls -l scripts/
total 20
-rw-rw-r-- 1 ubuntu02 ubuntu02  157  3월  2 15:51 coverage.sh
-rw-rw-r-- 1 ubuntu02 ubuntu02  163  3월  2 15:51 test.sh
-rw-rw-r-- 1 ubuntu02 ubuntu02  152  3월  2 15:51 test_contracts.sh
-rw-rw-r-- 1 ubuntu02 ubuntu02  149  3월  2 15:51 test_module.sh
-rw-rw-r-- 1 ubuntu02 ubuntu02 1625  3월  2 15:51 utils.sh
```

# Log
```
ubuntu02@ubuntu02-Z390-DESIGNARE:~/vagrant/npm-truffle-box$ npm run test

> @1.0.0 test /home/ubuntu02/vagrant/npm-truffle-box
> ./scripts/test.sh

sh: 1: ./scripts/test.sh: Permission denied
npm ERR! code ELIFECYCLE
npm ERR! errno 126
npm ERR! @1.0.0 test: `./scripts/test.sh`
npm ERR! Exit status 126
npm ERR!
npm ERR! Failed at the @1.0.0 test script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ubuntu02/.npm/_logs/2019-03-02T06_52_03_371Z-debug.log
```

--- a/package.json
+++ b/package.json
@@ -6,6 +6,7 @@
     "url": "git+https://github.com/wanseob/npm-truffle-box.git"
   },
   "scripts": {
+    "postinstall": "chmod +x scripts/*",
     "standard": "./node_modules/.bin/standard test/** src/** --fix",
     "ethlint": "./node_modules/.bin/solium -d contracts --fix",
     "precommit": "lint-staged && npm run test",
@@ -24,7 +25,8 @@
     "solium": "^1.2.2",
     "standard": "^12.0.1",
     "truffle": "^5.0.3",
-    "truffle-plugin-modularizer": "^1.2.3"
+    "truffle-plugin-modularizer": "^1.2.3",
+    "typedarray-to-buffer": "^3.1.5"
   },
   "dependencies": {
     "javascript-stringify": "^1.6.0",

